src/paradigma/imu_preprocessing.py

```diff
@@ -73,8 +72,8 @@ def preprocess_imu_data(input_path: str, output_path: str, config: IMUPreprocess
 def transform_time_array(
     time_array: np.ndarray,
     scale_factor: float,
-    input_unit_type: TimeUnit,
-    output_unit_type: TimeUnit,
+    input_unit_type: paradigma.TimeUnit,
```

Comment:
Should the type not be _str_?

Reply:
This applies to multiple edited lines in this file.

Reply:
TimeUnit is supposed to be an Enum (although I see it is not strictly one) with string values. So the type TimeUnit implies that only the values in the Enum are possible

Reply:
Thanks for clarifying that ;)